torch/_inductor/codegen/cuda/cuda_scheduling.py

```diff
@@ -41,3 +111,35 @@ def define_kernel(self, src_code: str, node_schedule) -> str:
                 kernel_name, compile_wrapper.getvalue(), metadata_comment
             )
         return kernel_name
+
+    def codegen_template(self, template_node, epilogue_nodes):
```

Comment:
The Triton version of codegen_template needed adaptation for our usecase here, so this is a modified version of that.